### DISCUSSION TITLE
Uploading crashes Cyberduck

### DISCUSSION THREAD

**Discussion by ronanmckenna-toolstop** [iterate-ch/cyberduck#14696] [Q&A]
Cyberduck works as normal until I try to upload a file to our server (WebDAV (HTTPS/SSL))

Im using Windows 10, the most recent Cyberduck - 8.5.9

Uninstalling and reinstalling does not resolve the issue. Neither does creating a new connection to the same location.

The problem started on 3rd of May. My colleagues older version works fine.

Any ideas?

**Comment by ylangisc** [comment]
Please try to empty all transfers in the Transfers window to resolve the issue.

**Reply by ronanmckenna-toolstop** [reply]
Thanks for the reply. 
Trying to open the Transfers window also crashes Cyberduck. That makes sense in regards to the crashing. 
Do you know of a way to delete the transfer info in the folder structure without opening the Transfers window on the interface?

**Reply by ronanmckenna-toolstop** [reply]
Solved. 
Delete the contents of `%AppData%\Cyberduck\Transfers`
